trie/triedb/pathdb/nodebufferlist.go

```diff
@@ -270,12 +270,18 @@ func (nf *nodebufferlist) flush(db ethdb.KeyValueStore, clean *fastcache.Cache,
 	}
 
 	commitFunc := func(buffer *multiDifflayer) bool {
+		if nf.count <= nf.rsevMdNum {
+			log.Info("Skip forcing flush buffer list due to buffer list is too less",
```

Comment:
I think this log is not accurate， maybe ("keep %d mulitidlfflayer in node bufferlist for getting withdrawal proof", nf.rsevMdNum) is better.

Reply:
fixed